docker-compose.yml

```diff
@@ -239,6 +239,11 @@ services:
       - './secrets/mirror-intel.env'
     environment:
       RUST_LOG: warn
+      http_proxy: http://172.17.0.1:8080
+      https_proxy: http://172.17.0.1:8080
+      HTTP_PROXY: http://172.17.0.1:8080
```

Comment:
If this refers to `tunnel`, you can refer it using just `http://tunnel:8080`

Reply:
This is host machine. We open a proxy on host to use IPv6.